about.html

```diff
@@ -0,0 +1,114 @@
+                <li class="nav-item">
+                    <a href="#" class="nav-link campaign">CC Campaign</a>
+                </li>
+            </ul>
+        </div>
+         
+       
+      </nav>
+      <section class="header">
+       <div class="header-1">
+        <p>Hi, Everyone</p>
+        <h1 class="header-title"> Digital Marketing Capstone Project: <br> A User-Friendly Website to Manage and Monitor Smart Strategies</h1>
+    </div>
+        <p class="para-2">The primary aim of this project is to develop a user-friendly website that allows businesses and marketers to manage and monitor their digital marketing strategies effectively and efficiently. This platform enables users to have centralized control over their marketing endeavors and leverage the power of social networks to expand brand awareness and increase conversions.</p>
+       
+       <div class="para-3-cont">
+       <p>Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+       <span><a href=""> ccsummit2015@cckorea.org</a></span>
+       </div>
+        <div class="cont-4">
+       <h2>Creative Commons <br> Global Summit 2015 Logo<h2>
+        <div class="separator"></div>
+        <p class="para-3">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+        <p class="para-8">The logo of Digital Marketing 2011 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+        </div>
+        <div class="logo-2"><img src="images/Market (1).png" alt=""></div>
+        
+      </section>
+
+      <section class="program-2">
+        <div class="title-3">See the past CC Global Summits</div>
+        <div class="separator"></div>
+        <p> Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw </p>
+
+        <div class="img-col">
+            <img src="images/summit-1.png" alt="">
+            <img src="images/summit-1 (1).png" alt="">
+        </div>
+    </section>
+
+    <footer>
+      <div class="footer">
+        <h1>Partner</h1>
+        <ul class="line-1">
+            <li><img src="images/partner/mozilla 1.png" alt=""></li>
+            <li><img src="images/partner/googlekorea 1.png" alt=""></li>
+            <li><img src="images/partner/naver 1.png" alt=""></li>
+        </ul>
+        <ul class="line-2">
+            <li><img src="images/partner/daumkakaologo 1.png" alt=""></li>
+            <li><img src="images/partner/airbnb 1.png" alt=""></li>
+        </ul>
+
+    </div>
+
+        <div class="copyrigth">
+            <img src="images/logo-white.png" alt=""> 
+            <p>(c)2015 D-Market. Some Rights Reserved.</p>
+        </div>
+      
+    </footer>
```

Comment:
- [ ] Kindly make sure that you use There's at least 1, but no more than 1, `h1` tag in each page, So Kindly **I recommend you to use `h3` tag for the Partner heading and make the partner as` section` tag instead of the footer page** , The Footer page should only contains **a logo aligned to the left and text aligned to the right.** Kindly Fix the issues in order to keep up the HTML hierarchy  and best practices Just like the below code snippets. 
```
<section  id="Partners">
        <div >
          <h3 >Partners</h3>
        </div>
        <ul >
          <li>
            <img >
          </li>
          <li>
            <img >
          </li>
          <li>
            <img >
          </li>
          <li>
            <img >
          </li>
          <li>
            <img >
          </li>
        </ul>
    </section>
 

   <footer >
        <div >
            <img >
        </div>
        <div  >
          <p></p>
        </div>
      </div>
   </footer>
```